peps/pep-2026.rst

```diff
@@ -2,11 +2,12 @@ PEP: 2026
 Title: Calendar versioning for Python
 Author: Hugo van Kemenade
 Discussions-To: https://discuss.python.org/t/pep-2026-calendar-versioning-for-python/55782
-Status: Draft
+Status: Rejected
 Type: Process
 Created: 11-Jun-2024
 Python-Version: 3.26
 Post-History: `14-Jun-2024 <https://discuss.python.org/t/pep-2026-calendar-versioning-for-python/55782>`__
+Resolution: https://discuss.python.org/t/pep-2026-calendar-versioning-for-python/55782/126
```

Comment:
Optionally use the short link?
```suggestion
Resolution: https://discuss.python.org/t/55782/126
```

Reply:
Thanks, I'll keep the long one, it's more descriptive and also used in Post-History.